### ISSUE TITLE
other task---edgenext

### ISSUE THREAD

**Issue by SDarknessZX** [mmaaz60/EdgeNeXt#9]
I used Edgenext for crowd counting, but the effect was not as good as VGG. I tried to change the different learning rate, but it still didn't help. Do you have any suggestions?

**Comment by mmaaz60** [comment]
Hi @SDarknessZX,

Thank you for using EdgeNeXt. We empirically found that EdgeNeXt is sensitive to hyperparameter choice for downstream vision tasks (Object detection and segmentation). We were, however, able to get good results when trained using a slightly less learning rate than usual.

Further, would you please tell me if crowd counting task is sensitive to location information? If that so, one thing to try could be to use learnable positional encodings in all SDTA blocks. Note that currently it is only enabled in the first SDTA block in the network. We observed that it was helpful but also increased the inference time a little. Please do share if you will have any questions or get any improvements in the crowd counting task.

Good Luck!

**Comment by SDarknessZX** [comment]
> Hi @SDarknessZX,
> 
> Thank you for using EdgeNeXt. We empirically found that EdgeNeXt is sensitive to hyperparameter choice for downstream vision tasks (Object detection and segmentation). We were, however, able to get good results when trained using a slightly less learning rate than usual.
> 
> Further, would you please tell me if crowd counting task is sensitive to location information? If that so, one thing to try could be to use learnable positional encodings in all SDTA blocks. Note that currently it is only enabled in the first SDTA block in the network. We observed that it was helpful but also increased the inference time a little. Please do share if you will have any questions or get any improvements in the crowd counting task.
> 
> Good Luck!

The crowd count is sensitive to the scale change, and the location also has a certain impact. I will try it according to your suggestion. Thank you!
